From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mina Her <minacle@live.com>
Date: Thu, 24 Sep 2026 14:24:46 +0900
Subject: [PATCH] Enable HEIF image loading

- Add `VipsForeignLoadHeif` to the configured libvips operations

Rationale: Registering the HEIF loader makes HEIF images available to applications using libvips.

diff --git a/config/initializers/vips.rb b/config/initializers/vips.rb
index 7799020..a539d70 100644
--- a/config/initializers/vips.rb
+++ b/config/initializers/vips.rb
@@ -18,6 +18,7 @@ if Rails.configuration.x.use_vips
     VipsForeignLoadJpeg
     VipsForeignLoadPng
     VipsForeignLoadWebp
+    VipsForeignLoadHeif
     VipsForeignSavePng
     VipsForeignSaveSpng
     VipsForeignSaveJpeg
